include/exadg/fluid_structure_interaction/precice/driver_fluid.h

```diff
@@ -218,7 +216,32 @@ class DriverFluid : public Driver<dim, Number>
 
       // post-solve
       if(is_new_time_window)
+      {
+        // computes new time-step size
         fluid->time_integrator->advance_one_timestep_post_solve();
+        // next, we synchronize the time-step sizes. Have a look in the base class for more
```

Comment:
Where does a synchronization happen?

Reply:
The `advance` call return the time remaining until the next time-window. The comment was rephrased anyway, let's see if it is more clear now.